src/test/FuzzerImpl.cpp

```diff
@@ -752,20 +759,9 @@ TransactionFuzzer::inject(std::string const& filename)
         return;
     }
 
-    // construct transaction
-    auto txFramePtr =
-        createFuzzTransactionFrame(mSourceAccountID, ops, mApp->getNetworkID());
-
-    if (Logging::logDebug("Ledger"))
-    {
-        LOG(DEBUG) << xdr_to_string(txFramePtr->getEnvelope());
-    }
-
     resetTxInternalState(*mApp);
     LedgerTxn ltx(mApp->getLedgerTxnRoot());
-
-    // attempt to apply transaction
-    txFramePtr->attemptApplication(*mApp, ltx);
+    attemptToApplyOps(ltx, mSourceAccountID, ops, *mApp, false);
```

Comment:
`attemptApplication` seems to do exactly what you wanted?

Reply:
`attemptApplication()` is a method of `FuzzTransactionFrame`, which we were creating in several places -- `attemptToApplyOps()` factors that out.  It also factors out the `throw` when application fails, which we want during setup (though not while running generated test cases).  Is there a better way of avoiding that duplication?